org.eclipse.lsp4e.test/src/org/eclipse/lsp4e/test/rename/RenameTest.java

```diff
@@ -103,10 +103,12 @@ public void testAsyncRenameHandlerEnablement() throws Exception {
 		ICommandService commandService = PlatformUI.getWorkbench().getService(ICommandService.class);
 		Command command = commandService.getCommand(IWorkbenchCommandConstants.FILE_RENAME);
 		assertFalse(command.isEnabled() && command.isHandled());
-
-		Thread.sleep(2 * delay);
-		assertTrue(command.isEnabled());
-		assertTrue(command.isHandled());
+		
+		Thread.sleep(delay * 3);
+		
+		// Put back so shutdown doesn't time out
+		MockLanguageServer.INSTANCE.setTimeToProceedQueries(0);
+		assertTrue(command.isEnabled() && command.isHandled());
```

Comment:
could you please keep the asserts separate:
```
assertTrue(command.isEnabled());
assertTrue(command.isHandled());
```

Reply:
Sorry - the email notification of your comment only arrived after I had independently fixed the build errors and @mickaelistria had merged... Please feel free to revert to two separate asserts.